services/web/client/source/class/qxapp/component/filter/TagsFilter.js

```diff
@@ -0,0 +1,126 @@
+/* ************************************************************************
+
+   qxapp - the simcore frontend
+
+   https://osparc.io
+
+   Copyright:
+     2019 IT'IS Foundation, https://itis.swiss
+
+   License:
+     MIT: https://opensource.org/licenses/MIT
+
+   Authors:
+     * Ignacio Pascual (ignapas)
+
+************************************************************************ */
+
+/**
+ * Dropdown menu for tag filtering
+ */
+
+qx.Class.define("qxapp.component.filter.TagsFilter", {
+  extend: qxapp.component.filter.UIFilter,
+
+  construct: function(filterId, groupId, labelTr = "Tags") {
+    this.base(arguments, filterId, groupId);
+    this._setLayout(new qx.ui.layout.HBox());
+
+    this.__dropDown = new qx.ui.toolbar.MenuButton(this.tr(labelTr));
```

Comment:
it is important to have `tr` around the actual string as this allows the compiler to pick out the strings for translation and add the `.po` files.